tests/modules/vsearch/usearchglobal/test.yml

```diff
@@ -0,0 +1,17 @@
+- name: vsearch usearchglobal test_vsearch_usearchglobal
+  command: nextflow run ./tests/modules/vsearch/usearchglobal -entry test_vsearch_usearchglobal -c ./tests/config/nextflow.config  -c ./tests/modules/vsearch/usearchglobal/nextflow.config
+  tags:
+    - vsearch/usearchglobal
+    - vsearch
+  files:
+    - path: output/vsearch/test.aln
+      md5sum: 2543c4147530dcb1ba4550d3fdb1502a
```

Comment:
That is unfortunate. In the worst case, you can only verify that the files exists by omitting the whole md5sum line. However, this isnt recommended, see 

> 5. [./tests/modules/fastqc/test.yml](https://github.com/nf-core/modules/blob/master/tests/modules/fastqc/test.yml)
This file will contain all of the details required to unit test the main script in the point above using [pytest-workflow](https://pytest-workflow.readthedocs.io/). If possible, any outputs produced by the test workflow(s) MUST be included and listed in this file along with an appropriate check e.g. md5sum. The different test options are listed in the [pytest-workflow docs](https://pytest-workflow.readthedocs.io/en/stable/#test-options).

in https://nf-co.re/developers/modules#new-module-workflow. But `pytest-workflow docs` (as above) specify also e.g.
```
      contains:                      # A list of strings that should be in the file (optional)
        - "moo"
```
maybe its possible to get there some lines to test?

Maybe one of the output formats do not have such resource-specific strings (maybe the biom table)? Than it might make sense to only check for existence with the first two tests and check the md5sum for one additional file?